### PR TITLE
 On branch data-structure_array

### PR DESCRIPTION
**Pull request by ryoasaka12345**
Changes to be committed:
	new file:   DataStructure/array/hw1.php
	new file:   DataStructure/array/hw2.php
	new file:   DataStructure/array/hw3.php

### CODE REVIEW

File: DataStructure/array/hw1.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+echo "<br>";
+
+// Ttest3
+$arr3 = [9, 5, 3, 7, 21];
+$Arr3 = new CalcValues($arr3);
+$Arr3->echoArray();
+$Arr3->echoValues();
+echo "<br>";
+
+?>
```
Comment (pav-thanhtt) [inline DataStructure/array/hw1.php:127]: how about?
```
$arr4 = [];
$Arr4 = new CalcValues($arr4);
$Arr4->echoArray();
$Arr4->echoValues();
echo "<br>";
```

File: DataStructure/array/hw2.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+$arrayA = [1, 2, 3, 4, 5];
+$arrayB = [4, 5, 3, 2, 10];
```
Comment (pav-thanhtt) [inline DataStructure/array/hw2.php:4]: how about?
```
$arrayA = [1, 2, 3, 4, 5];
$arrayB = [4, 5, 3, 2, 10, 15, 20];
```

File: DataStructure/array/hw3.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+$array = [
+    [5, 12, 17, 9, 13],
+    [13, 4, 8, 14, 1],
+    [9, 5, 3, 17, 21]
+];
+
```
Comment (pav-thanhtt) [inline DataStructure/array/hw3.php]: how about 
`$array = []`;

File: DataStructure/array/hw1.php [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        if ( gettype($arr) != "array" || $arr == []) {
+            echo "<br>";
+            echo "Incorrect input";
+            $this->arr = [];
+            return; 
+        }
+        
+        $this->arr = $arr;
+        $this->size = sizeof($arr);
+        $this->sum = 0;
```
Comment (pa-vuhn) [inline DataStructure/array/hw1.php:35]: It's very good to exercise OOP here :)
Usually, we don't put all logic into constructor. Only init some value, please move calculation to a function

**Pull request merged**